### PR TITLE
Add video embed support for updates

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/4298

Depends on: https://github.com/opencollective/opencollective-api/pull/6067

**Screen-cast**

https://user-images.githubusercontent.com/12435965/119249140-3620e980-bb4b-11eb-8c9d-5745806c0a43.mp4

### CODE REVIEW

**Comment by znarf** [comment]
@Betree can we ship this?

**Comment by Betree** [comment]
@znarf It depends on https://github.com/opencollective/opencollective-frontend/pull/6327, we'll need to merge it first

File: pages/createUpdate.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ class CreateUpdatePage extends React.Component {
                 </Container>
               </Link>
             </BackButtonWrapper>
-            <Container width={1} maxWidth={650}>
+            <Container width={1} maxWidth={680}>
```
Comment (Betree) [inline pages/createUpdate.js:113]: @SudharakaP Was this part of a design update? Is it required for video embedding?

Comment (SudharakaP) [reply pages/createUpdate.js:149]: Addition of the video button on the rich text editor toolbar make it like this on the updates page; 😉 

image

So we need to change the width a bit.

Comment (Betree) [reply pages/createUpdate.js:149]: Ok, this might have an impact on the responsiveness but let's not block on this

**Review by Betree — COMMENTED** [review]
Let's make sure we merge and deploy https://github.com/opencollective/opencollective-api/pull/6067 first

**Review by Betree — APPROVED** [review]

File: pages/createUpdate.js (hunk 1 above)
Comment (SudharakaP) [reply pages/createUpdate.js:149]: I tested this on various screen sizes and I don't see an apparent loss of responsiveness; 

Screen Shot 2021-11-17 at 11 28 32

If you see any issues let me know I'll open a new issue to fix that. 😉

**Pull request merged**